ga4gh/protocol.py

```diff
@@ -4,6 +4,7 @@
 from __future__ import division
 from __future__ import print_function
 from __future__ import unicode_literals
+from itertools import imap
```

Comment:
Arbitrary nitpick: I think of the import section as a bunch of groups: (1) the `__future__` section; (2) the standard library; (3) third party libraries; (4) within package imports. I think it's useful to group them this way so we can see what's coming from where. I also like to order the imports within each group in increasing order of line-length. I find it useful, but it might just be an embarrassing aesthetic habit on my part!

I would tend to prefer importing  a module and using fully referenced names in most cases too. I think this makes it clearer what's happening when reading the code. I might not immediately remember what imap does in this case when I'm looking at the code below, whereas `itertools.imap` tells me what sort of thing it's likely to do.

These are all very subjective style issues, and I wouldn't argue in favour of any of them strongly. It would be useful to come up with some convention on imports though, and to document this in our style guide.